app/workers/publishing_api_worker.rb

```diff
@@ -39,4 +44,13 @@ def save_draft_of_unpublished_edition(unpublishing)
       Whitehall::PublishingApi.save_draft_async(draft)
     end
   end
+
+  def handle_error(error)
+    if error.code >= 500
+      raise error
+    else
+      explanation = "The message is a duplicate and does not need to be retried"
+      Airbrake.notify_or_ignore(error, parameters: { explanation: explanation })
```

Comment:
@boffbowsh I've just checked with @tuzz the author of the [commit](https://github.com/alphagov/publishing-api/commit/3dd1b6bae628e32de477fc797e4c0666dad2827e) this work is based on.
The behaviour should be that rescuing and re-raising an error adds a job to the retry queue, whereas rescuing and notifying Airbrake won't add a retry.
Apologies for the poor description, noise won't be reduced as you mention, notifications are still sent via Airbrake.